spec/controllers/workflows_controller_spec.rb

```diff
@@ -22,9 +22,32 @@
     end
 
     it 'serialized transcription groups' do
-      allow_any_instance_of(Workflow).to receive(:groups).and_return({"FIRST" => 2, "SECOND" => 1})
+      # TODO: the results here look out of sync with the actual group data from the workflow
```

Comment:
Also want to remove these lines and open an issue if necessary. TODOs are not tracked on this repo.

Reply:
Ah, heck. I fixed the out-of-sync data, just forgot to remove the comment.